tests/schedule_test.go

```diff
@@ -1240,3 +1191,13 @@ func (s *ScheduleFunctionalSuite) refreshWorkerServices() {
 		w.RefreshPerNSWorkerManager()
 	}
 }
+
+func (s *ScheduleFunctionalSuite) cleanup(sid string) {
```

Comment:
I saw that the cleanup was removed from a few of the tests; when's cleanup needed for these tests? It looks like it's being used whenever we want to assert that a workflow's function was called?

Reply:
I think all these tests should attempt to clean up after themselves (though they should also accept other schedules running in parallel without getting confused).

I just moved these DeleteSchedule calls into t.Cleanup so they run even if the test fails. And factored it out into a function on the suite. Actually I can do a couple more...